torch/csrc/jit/codegen/cuda/iter_visitor.h

```diff
@@ -205,6 +214,17 @@ class TORCH_CUDA_API DependencyCheck {
       const std::vector<Val*>& of);
 };
 
+// Simply grabs all exprs needed to produce provided outputs, only in dependency
+// order, not computeAtOrder like ExprSort in fusion.h
+class UnsortedExprs : public IterVisitor {
```

Comment:
`ExprSort` doesn't do anything for computeAt anymore as the logic is moved to lower_loops.cpp. Perhaps, we don't need this class and can just use `ExprSort`.